index.html

```diff
@@ -27,9 +27,6 @@ <h1>Tony's House of Tapes</h1>
             <h3>We have your solution</h3>
             <h1>Tapes!</h1>
         </section>
-        <section>
-          <img src="images/1024px-Compactcassette.jpg" alt="">
```

Comment:
for accessibility, you can always include an "alt" by defaulting to the name of the image or a description as a string which a screen reader would pick up.

Reply:
Thanks, I'll keep that in mind, are there any good resources on accessibility to look at?